index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+    <div class="Header">
+        <h1  class="mylogo" >MY logo</h1>
+        <div>
+            <img src="./image/logobuger.JPG">
+        </div>
+    </div>
+<section class="main-section">
+    
+        <h1 class="main-header"> I'm Mariane Glad to see you!</h1>
+    
+        <p class = "home-text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+        <div class = "social-section">
+            <h4 class="connect"> LET’S CONNECT</p> 
+            <ul class="social-icons">
+                <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+            </ul>
+    
+        </div>
+
+</section>
+</body>
+</html>
```

Comment:
- [ ] :red_circle: As requested by the previous reviewer, please make sure your site is responsive. As for now, the horizontal scroll is still there 
[requesed.webm](https://user-images.githubusercontent.com/26736582/198821947-4bdba0c5-7b03-4792-a782-f846754e27a1.webm)